### DISCUSSION TITLE
Add Rss feed link

### DISCUSSION THREAD

**Discussion by ririko5834** [upptime/upptime#198] [Ideas]
Can you please add RSS feed feature, where will be all incidents added? So members can subscribe to this feed to be notified on new incidents.

**Comment by Nirmitjatana** [comment]
@AnandChowdhary would you like this to be implemented?

**Reply by AnandChowdhary** [reply]
For sure! Happy to accept a PR!

**Reply by leggetter** [reply]
@AnandChowdhary - any hints on how a contributor could go about adding this?

I'd guess it's a matter of generating an `rss.xml` file whenever a site update takes place. If you can provide some pointers I'm happy to take a look.
